backend/services/theme_service.py

```
@@ -77,97 +77,343 @@ def _create_smart_person_background_mask(self, image_bytes: bytes, image_width:
+        Returns:
+            bytes: Color-harmonized image with natural face-body blending
+        """
+        try:
+            # Load images
+            generated_image = Image.open(io.BytesIO(generated_image_bytes)).convert('RGB')
+            original_image = Image.open(io.BytesIO(original_image_bytes)).convert('RGB')
+            mask_image = Image.open(io.BytesIO(mask_bytes)).convert('L')
+            
+            # Convert to numpy arrays for processing
+            generated_array = np.array(generated_image)
+            original_array = np.array(original_image)
```

Ruff (0.12.2)

Local variable `original_array` is assigned to but never used

Remove assignment to unused variable `original_array`

(F841)